src/bot.js

```diff
@@ -7,14 +7,15 @@ const ConfigHandler = require('./handlers/config-handler');
 const CommandHandler = require('./handlers/command-handler');
 const ModuleHandler = require('./handlers/module-handler');
 const LowDBHandler = require('./handlers/lowdb-handler');
+const NeDBHandler = require('./handlers/nedb-handler');
 
 const bot = new discordjs.Client();
 bot.loggingHandler = new LoggingHandler(bot);
 bot.config = new ConfigHandler(bot);
 bot.commandHandler = new CommandHandler(bot);
 bot.moduleHandler = new ModuleHandler(bot);
 bot.db = new LowDBHandler(bot, path.resolve(__dirname, '../data'));
-
+bot.db2 = new NeDBHandler(bot, path.resolve(__dirname, '../data2'));
```

Comment:
While good for demonstrating it's use, ultimately this would be only one DB in use at a time. I guess for now keep this, but it'll get removed after I try out NeDB and decide if I like it or not :)